src/test/resources/mockdata/compositeOrders/1ab7ef6a-d1d4-4a4f-90a2-882aed18af14.json

```diff
@@ -122,7 +122,7 @@
       "poLineNumber": "268758-01",
       "publicationDate": "2011",
       "publisher": "Tapir Academic Press",
-      "purchaseOrderId": "d79b2bcc-DcAD-1E4E-Abb7-DbFcaD5BB3bb",
```

Comment:
I can't believe we're still sorting these out, but I'm glad to see you took the time to do so.